### PR TITLE
Appjupyter

### LINKED ISSUE #140: [SDKTECHNO-56] Add jupyter notebook
Integrate Jupyter Notebook for Saagie into Technologies repository : https://github.com/saagie/jupyter-python-nbk

### PR DESCRIPTION
**Pull request by finalspy**
Closes #140

### CODE REVIEW

**Comment by finalspy** [comment]
auto-sklearn lib has been removed temporarily because it leads the build to fail due to : 
```
DEPRECATION: Could not build wheels for auto-sklearn, ConfigSpace which do not use PEP 517. pip will fall back to legacy 'setup.py install' for these. pip 21.0 will remove support for this functionality. A possible replacement is to fix the wheel build issue reported above. You can find discussion regarding this at https://github.com/pypa/pip/issues/8368.
```

**Comment by finalspy** [comment]
@PierreLeresteux you can review this one, tests are OK for me.
May be someone from the service could test and review it as well ?

File: technologies/app/jupyter/jupyter-minimal/image_test.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    command: "pwd"
+    expectedOutput: ["/notebooks-dir"]
+  - name: "curl"
+    args: ["--help"]
+    command: "curl"
+    exitCode: 0
+  - name: "Check conda virtual env"
+    command: "bash"
+    args: ["-c", "conda env list | grep py36"]
+    expectedOutput: ["py36                     /opt/conda/envs/py36"]
```
Comment (PierreLeresteux) [inline technologies/app/jupyter/jupyter-minimal/image_test.yaml:27]: j'aurai plutôt mis ça : 
`expectedOutput: ["/opt/conda/envs/py36"]`

car le nombre d'espace c'est pas terrible dans le test non ?

**Comment by finalspy** [comment]
@PierreLeresteux tests updated.
Review par service  OK

**Review by PierreLeresteux — APPROVED** [review]

**Pull request merged**